judo-runtime-core-bootstrap/src/main/java/hu/blackbelt/judo/runtime/core/bootstrap/dao/rdbms/RdbmsBuilderProvider.java

```diff
@@ -67,6 +74,16 @@ public class RdbmsBuilderProvider implements Provider<RdbmsBuilder> {
     @Inject
     Dialect dialect;
 
+    @Inject(optional = true)
+    @Named(RDBMS_DAO_FLOATING_POINT_TYPE_MAX_PRECISION)
+    @Nullable
+    private int precision = 15;
+
+    @Inject(optional = true)
+    @Named(RDBMS_DAO_FLOATING_POINT_TYPE_MAX_SCALE)
+    @Nullable
+    private int scale = 4;
```

Comment:
Why 4? Above we declared 7 as max scale.

Reply:
7 was the wrong declaration.

Reply:
[ace61b5](https://github.com/BlackBeltTechnology/judo-runtime-core/pull/154/commits/ace61b5646a8cc193b84ac4212c453d9d95678a9)